### ISSUE TITLE
BeanValidation does not work in nested objects or arrays

### ISSUE THREAD

**Issue by schlagi123** [openapi-processor/openapi-processor-core#64]
Hi,

the BeanValidation is not working if I create a Model with nestead structure. This is because Objects or Lists need a @Valid annotation.

**Comment by hauner** [comment]
Hi,

yes, the array/collection case is missing.

can you please provide an example for the `object` case. There is a test case with a nested object and it passes. There is probably a detail that is not correctly handled.

**Comment by hauner** [comment]
available in 2021.4-SNAPSHOT, you can test it by adding it to the processor configuration.

Example:

  ```groovy
  // ...
  spring {
     processor 'io.openapiprocessor:openapi-processor-core:2021.4-SNAPSHOT'
     processor 'io.openapiprocessor:openapi-processor-spring:2021.3.1'
     // ...
  }
  ```
